src/services/cache/CacheService.ts

```diff
@@ -79,6 +92,123 @@ export class CacheService {
     return this.dmrService.readResource(request);
   }
 
+  getCacheStats(container: ICacheContainer, cache: ICache): ng.IPromise<any> {
+    let firstServer: IServerAddress = container.serverGroup.members[0];
+    let address: string [] = [].concat("host", firstServer.host, "server", firstServer.name, "subsystem",
+      "datagrid-infinispan", "cache-container", container.name, cache.type, cache.name);
```

Comment:
O yeah, I missed that, maybe we should create a generic function to create the server path then? Also, does it matter which node we get the stats from? AFAIK, it shouldn't but wasn't sure if going via the coordinator would make any difference.